_layouts/element.html

```diff
@@ -16,7 +16,12 @@ <h1 class="pldoc-heading pldoc-element-title hd-4">{{ page.title }}</h1>
                 </li>
                 {% if page.example %}
                 <li class="meta meta-examplelink">
-                    <a class="pldoc-link" href="{{ site.exampleurl }}{{page.example}}"><span class="sr-only">View</span>Detailed Example</a>
+                    <a class="pldoc-link" href="{{ site.exampleurl }}{{ page.example }}"><span class="sr-only">View</span>Detailed Example</a>
```

Comment:
Do you need a space in the span with "View" so it doesn't run together with "Detailed"?